src/app/admin/participants/page.js

```diff
@@ -1,14 +1,23 @@
 "use client";
 import Participants from "@/components/Participants";
 import React from "react";
+import { useSession } from "next-auth/react";
+import ProtectedPage from "@/components/ProtectedPage";
 
 const ParticipantsPage = () => {
-  return (
-    <div className="w-11/12">
-      <title>Admin | Participants</title>
-      <Participants />
-    </div>
-  );
+  const session = useSession();
+  console.log(session);
+
+  if (session?.data?.user?.role === "admin") {
```

Comment:
lol what is this 😭 the protected page handles this internally, u don't need the admin check